src/main/java/redis/clients/jedis/resps/KeyedZSetElement.java

```diff
@@ -0,0 +1,43 @@
+package redis.clients.jedis.resps;
```

Comment:
I am not sure whether putting in `redis.clients.jedis.resps` this package is a good solution or not, currently lots of model class in `redis.clients.jedis` such as `StreamPendingSummary`、`Tuple`

Reply:
We can move the new ones (targeted 3.6.0) here and plan to gradually move the existing ones.

Reply:
👍 Is it also possible to add `package-info.java` to let people know what it's for?

Reply:
sure!

Reply:
@dengliming package-info added.